contributor-docs/CONTRIBUTING.md

```diff
@@ -43,10 +43,10 @@ As this is a public repo, please be careful not to include details or screenshot
 
 We primarily use our documentation site as a workspace to develop new components or make changes to existing components (stay tuned for a better development environment coming soon!).
 
-Before running the documentation site locally, you'll need to install packages in the root and `docs` directories:
+Before running the documentation site locally, make sure to install [Node.js](https://nodejs.org/en/) v16 (we recommend using [nvm](https://github.com/nvm-sh/nvm)). Next, install packages in the root and `docs` directories:
 
 ```sh
-npm install && cd docs && npm install
+npm install && cd docs && npm install --legacy-peer-deps
```

Comment:
I forgot to mention that we have a `setup` script. Let's recommend people use that instead:
 
```suggestion
npm run setup
```

We'll also need to add the `--legacy-peer-deps` flag to the setup script: https://github.com/primer/react/blob/main/script/setup#L11

Reply:
@colebemis sounds good! Just updated